commit c956aa8c4e9a594eb44737c907ed5f9d06d0a26c (refs/changes/77/1877/4)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-05-10T22:03:36+00:00 (2 years, 5 months ago)
    
    PUBAPI-1389 cloudapi should remove RC4, 3DES
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/etc/stud.conf b/etc/stud.conf
index 35d3ca6..10e8447 100644
--- a/etc/stud.conf
+++ b/etc/stud.conf
@@ -34,9 +34,9 @@ pem-file = "/opt/smartdc/cloudapi/ssl/stud.pem"
 # Run openssl ciphers for list of available ciphers.
 # type: string
 #
-# Preferred cipher list taken from
-# https://github.com/cloudflare/sslconfig
-ciphers = "EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:HIGH:RC4-SHA:!MD5:!aNULL:!PSK"
+# Preferred cipher list as per RFD 93
+# https://github.com/joyent/rfd/tree/master/rfd/0093
+ciphers = "EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:HIGH:!MD5:!aNULL:!PSK"
 
 # Enforce server cipher list order
 #
@@ -100,7 +100,8 @@ daemon = on
 
 # Report client address by writing IP before sending data
 #
-# NOTE: This option is mutually exclusive with option write-proxy and proxy-proxy.
+# NOTE: This option is mutually exclusive with option write-proxy and
+# proxy-proxy.
 #
 # type: boolean
 write-ip = off
